StatBot/Cards/StatisticButton.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Bot.Schema;
+using StatBot.Interfaces;
+using StatBot.Models;
+
+namespace StatBot.Cards
+{
+    public class StatisticButton : ICard
+    {
+        public CardAction Action { get; set; }
+        public List<string> ButtonsName { get; set; }
+
+        public Activity OnClick(Activity activity)
+        {
+            if (activity?.Conversation != null)
+            {
+                activity.Attachments = new List<Attachment>();
+                Attachment attachment;
+                var actions = new List<CardAction>();
+
+                var data = activity.Text.Split(' ');
+                string CommandType = data[2];
+                int StatisticId = Convert.ToInt32(data[1]);
+
+                if (CommandType == "TurnOffOn")
+                {
+                    actions.Add(new ActionButton(StatisticId, "Включить статистику", "SwitchOn").Action);
+                    actions.Add(new ActionButton(StatisticId, "Выключить статистику", "SwitchOff").Action);
+                }
+                else if (CommandType == "Subs")
+                {
+                    actions.Add(new ActionButton(StatisticId, "Подписаться", "Subscribe").Action);
+                    actions.Add(new ActionButton(StatisticId, "Отписаться", "Unsubscribe").Action);
+                }
+                Attachment a = new Attachment()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'a' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS1481)